master/pkg/model/task.go

```diff
@@ -85,6 +85,8 @@ type Task struct {
 	State *TaskState `db:"task_state" bun:"task_state"`
 
 	Config *string `db:"config"`
+
+	NoPause *bool `db:"no_pause"`
```

Comment:
the intention was to stop users from pausing child tasks [unless they really want to], and to only allow pausing root tasks by default. could we add a logic which defaults this to true for child task, unless a user overrides it in the API call?

i.e. if no_pause is set in the parameters, respect that
if no_pause is null and it's a root task, default false
if no_pause is null and it's a child task, default true

Reply:
Would this also mean that no_pause would be true by default in the DB, instead of false?

Reply:
since it depends on being root/child, there's no universal default value, so DB default can be either. I don't have a strong preference, `false` seems fine to me.